InputfieldChosenSelectMultiple.module

```diff
@@ -20,15 +20,15 @@ class InputfieldChosenSelectMultiple extends InputfieldSelectMultiple implements
 
 	protected $chosenOptions = array();
 	protected $supportedAdminThemes = array(
-		"AdminThemeReno", 
+		"AdminThemeReno",
 		"AdminThemeDefault"
 	);
 
 	public static function getModuleInfo() {
 		return array(
 			'title' => __('Inputfield Chosen Select Multiple', __FILE__), // Module Title
 			'summary' => __('Inputfield for the ProcessWire FieldtypePage. Provides an alternative tagging interface.', __FILE__), // Module Summary
-			'version' => 120, 
+        'version' => 121,
```

Comment:
Always nice to see the version bumped as well

Reply:
Wanted to make it easy for you :)